v2.2/cluster-setup-troubleshooting.md

```diff
@@ -6,202 +6,433 @@ toc: true
 
 If you're having trouble starting or scaling your cluster, this page will help you troubleshoot the issue.
 
-## Before you begin
-
-### Terminology
-
 To use this guide, it's important to understand some of CockroachDB's terminology:
 
   - A **Cluster** acts as a single logical database, but is actually made up of many cooperating nodes.
   - **Nodes** are single instances of the `cockroach` binary running on a machine. It's possible (though atypical) to have multiple nodes running on a single machine.
 
-### Using this guide
+## Cannot run a single-node CockroachDB cluster
 
-To diagnose issues, we recommend beginning with the simplest scenario and then increasing its complexity until you discover the problem. With that strategy in mind, you should proceed through these troubleshooting steps sequentially.
+Try running:
 
-We also recommend executing these steps in the environment where you want to deploy your CockroachDB cluster. However, if you run into issues you cannot solve, try the same steps in a simpler environment. For example, if you cannot successfully start a cluster using Docker, try deploying CockroachDB in the same environment without using containers.
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach start --insecure --logtostderr
+~~~
 
-## Locate your issue
+If the process exits prematurely, check for the following:
 
-Proceed through the following steps until you locate the source of the issue with starting or scaling your CockroachDB cluster.
+### An existing storage directory
 
-### 1. Start a single-node cluster
+When starting a node, the directory you choose to store the data in also contains metadata identifying the cluster the data came from. This causes conflicts when you've already started a node on the server, have quit cockroach, and then tried to start another cluster using the same directory. Because the existing directory's cluster ID doesn't match the new cluster ID, the node cannot start.
```

Comment:
"have quit cockroach" - cockroach lower case like this should usually be written as `cockroach` and typically refers to the process, so you'd say "have quit the `cockroach` process" or "have stopped CockroachDB"

Reply:
Excellent catch. Thanks!